advanced/Scripts/piholeARPTable.sh

```diff
@@ -54,6 +56,20 @@ flushARP(){
         return 1
     fi
 
+    # Flush ARP cache of the host
```

Comment:
Good catch.

I really don't know why it was removed and reintroduced, but if we really don't needed to flush the ARP cache, maybe we should change the function name from `flushARP` to `truncateARPtable` or something similar.

Reply:
Thanks for digging this out. I will be honest right away: I don't remember what triggered me to remove the ARP flushing before. However, I think it is very useful because if there is anything wrong in the hosts's ARP cache, then it will stay there and FTL will immediately re-import it from there even after the Pi-hole-internal ARP table has been flushed.

I suppose the bad consequences we have talked about in this commit were that the server shortly becomes quite disoriented inside the local network. It may take up to a few seconds until it rediscovered its neighbors in the network and can continue to use the network as usual. I just tried this on my local machine (having Wireshark opened at the same time to watch ARP traffic) and - for me - it took about 8 milliseconds until my server rediscovered the local gateway and was again able to talk to the Internet. Everything in between failed to be sent.

Reply:
The issue was with the mock devices of the network table.
https://discourse.pi-hole.net/t/request-ability-to-add-clients-to-known-client-list-from-network-page/28601/86?u=yubiuser